infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -193,8 +193,8 @@ public SequencerServer(ServerContext serverContext) {
     * @param conflictParam The conflict parameter.
     * @return A conflict hash code.
     */
-    public int getConflictHashCode(UUID streamId, int conflictParam) {
-        return Objects.hash(streamId, conflictParam);
+    public String getConflictHashCode(UUID streamId, int conflictParam) {
+        return streamId.toString() + Integer.toString(conflictParam);
```

Comment:
I think that by default, toStrong() prints the Java hashcode of the object, so this brings us back to where we started, no? 

if the intention is that this is just one step towards the full solution, I buy it.